setup.py

```diff
@@ -24,5 +24,5 @@
     author_email='oscm@edx.org',
     license='Apache',
     packages=find_packages(exclude=['*.tests']),
-    install_requires=['slumber', 'PyJWT'],
+    install_requires=['slumber >= 0.7.1', 'PyJWT >= 1.1.0'],
```

Comment:
@dan-f ordinarily I would agree with your proposed limits; however, `PyJWT` has a history of not properly following semantic versioning. This cause us problems a few months ago.